mods/rv/rules/allied-structures.yaml

```diff
@@ -1555,7 +1564,7 @@ gaapad:
 	Buildable:
 		Queue: Building.Allies
 		BuildPaletteOrder: 100
-		Prerequisites: gaairc, ~prerequisite.airpad, ~techlevel.2
+		Prerequisites: gaairc, ~techlevel.2, ~prerequisite.aiplayer #~prerequisite.airpad ## This can make AI build more aircraft depends on resupply
```

Comment:
I'm not sure if i like this idea. But i'll ask Ganon too.

Reply:
removed AI build but change here is not removed since it is unused for both AI and normal gameplay